src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,316 @@
+and give feedback on their usability, practicality, and effectiveness, both
+to us, and to the web standards community. For more information, see the
+[Origin Trials Guide for Web Developers][ot-guide].
+
+To enable the origin trial:
+
+1. Request a [token][ot-link] for your origin.
+2. Add the token to your pages, there are two ways to provide this token on
+   any pages in your origin:
+     * Add an `origin-trial` `<meta>` tag to the head of any page. For example,
+       this may look something like: <br>
+       `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+     * If you can configure your server, you can also provide the token on
+       pages using an `Origin-Trial` HTTP header. The resulting response header
+       should look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+### Feature detection
+
+To check if the Contact Picker API is supported, use:
+
+```js
+const supported = ('contacts' in navigator && 'ContactsManager' in window);
+```
+
+In addition, on Android, the Contact Picker requires Android M or later.
+
+### Opening the Contact Picker
+
+The entry point to the Contact Picker API is `navigator.contacts.select()`.
+When called, it returns a promise and shows the contact picker, allowing the
+user to select the contact(s) they want to share with the site. After
+selecting what to share and clicking **Done**, the promise resolves with an
+array of contacts selected by the user.
+
+When calling `select()` you must provide an array of properties you'd like
+returned as the first parameter, and optionally whether multiple contacts can be
+selected as a second parameter.
+
+```js
+const props = ['name', 'email', 'tel'];
+const opts = {multiple: true};
+
+try {
+  const contacts = await navigator.contacts.select(props, opts);
+  handleResults(contacts);
+} catch (ex) {
+  // Handle any errors here.
+}
+```
+
+The Contacts Picker API can only be called from a [secure][secure-contexts],
+top-level browsing context, and like other powerful APIs, it requires a
+user gesture.
+
+### Handling the results
+
+The Contact Picker API returns an array of contacts, and each contact
+includes an array of the requested properties. If a contact doesn't have
+data for the requested property, or the user chooses to opt-out of sharing
+a particular property, it returns an empty array.
+
+For example, if a site requests `name`, `email`, and `tel`, and a user
+selects a single contact that has data in the name field, provides two
+phone numbers, but does not have an email address, the response returned will be:
+
+```json
+[{
+  "email": [],
+  "name": ["Queen O'Hearts"],
+  "tel": ["+1-206-555-1000", "+1-206-555-1111"]
+}]
+```
+
+{% Aside 'caution' %}
+  Labels and other semantic information on contact fields are dropped.
+{% endAside %}
+
+## Security and permissions {: #security-considerations }
+
+We've designed and implemented the Contact Picker API using the core
```

Comment:
```suggestion
The Chrome team designed and implemented the Contact Picker API using the core
```

Avoiding "we" per Rob's rationale in https://github.com/GoogleChrome/web.dev/pull/1614/files#r333143069